Lucidtech/Las/Client.cs

```diff
@@ -527,6 +729,69 @@ public object CreateBatch(string description)
             return ExecuteRequestResilient(RestSharpClient, request);
         }
 
+        /// <summary>List executions in a transition, calls the GET /transitions/{transitionId}/executions endpoint.</summary>
+        /// <example>
+        /// <code>
+        /// Client client = new Client();
+        /// var response = client.ListTransitionExecutions("&lt;transitionId&gt;", new [] {"succeeded", "failed"});
+        /// </code>
+        /// </example>
+        /// <param name="transitionId">Id of the transition</param>
+        /// <param name="statuses">List of execution statuses to filter by</param>
+        /// <param name="executionIds">List of execution ids to filter by</param>
+        /// <param name="maxResults">Maximum number of results to be returned</param>
+        /// <param name="nextToken">A unique token used to retrieve the next page</param>
+        /// <param name="sortBy">The sorting variable of the execution: "endTime" | "startTime"</param>
+        /// <param name="order">Order of the executions: "ascending" | "descending"</param>
+        /// <returns>Transition executions response from the REST API</returns>
+        public object ListTransitionExecutions(
+            string transitionId,
+            List<string>? status = null,
+            List<string>? executionIds = null,
+            int? maxResults = null,
+            string? nextToken = null,
+            string? sortBy = null,
+            string? order = null
+        ) {
+            var queryParams = new Dictionary<string, object> {
+                {"status", status},
+                {"executionId", executionIds},
```

Comment:
Does this work? will the query-parameters be appended as expected? 
`?executionId=a&executionId=b....`